datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/metadata.py

```diff
@@ -277,6 +279,9 @@ def metadata(check, check_duplicates, show_warnings):
         duplicate_description_set = set()
 
         metric_prefix_error_shown = False
+        if os.stat(metadata_file).st_size == 0:
+            errors = True
+            echo_failure(f"{current_check} metadata file is empty")
```

Comment:
Minor suggestion to make the error more actionable, WDYT?
```suggestion
            echo_failure(f"{current_check} metadata file is empty. This file needs the header row at minimum")
```